app/views/pages/moving-guide/reimbursements/reimbursements-for-civilians.html.erb

```diff
@@ -0,0 +1,187 @@
+    <li><%= link_to 'Mobile Home Transportation', '#mobile-home-transportation' %></li>
+    <li><%= link_to 'Relocation Income Tax (RIT)', '#relocation-income-tax' %></li>
+    <li><%= link_to raw("Waive Limitation on #{abbr_tag('pcs')} Relocation Allowances"), '#waive-limitation-on-pcs-relocation-allowances' %></li>
+    <li><%= link_to raw("#{abbr_tag('pcs')} Allowances Time Limitation"), '#pcs-allowances-time-limitation' %></li>
+    <li><%= link_to 'Pets', '#pets' %></li>
+    <li><%= link_to 'Miscellaneous Expense Allowance (MEA)', '#miscellaneous-expense-allowance' %></li>
+  <% end %>
+
+  <div class="sidenav-body">
+    <p>When you travel on orders, the government may pay for many of your out of pocket expenses, either as a reimbursement after the fact, or as an advance. Reimbursable allowances will need to be listed and itemized on your voucher (DD Form 1351-2) and can be submitted by your local finance office. Furthermore, certain offices allow you to submit your voucher online, so it’s best to ask your finance office to see what works best for you.</p>
+    <p><b>Disclaimer:</b> Most dollar limits on civilian entitlements are imposed either by Federal Travel Regulation (FTR)(41 CFR 302) or Joint Travel Regulations (JTR), Volume 2. A notable exception are the Real Estate Expenses limitations. <%= abbr_tag('jtr') %> is DoD’s supplement to <%= abbr_tag('ftr') %>.</p>
+
+    <div id="advances">
+      <h2>Advances</h2>
+
+      <p>Since you have to pay for some things up front, then file a voucher <em>and</em> you anticipate that your finances will be strained during your <%= abbr_tag('pcs') %> move, you can request a monetary “Advance” to help cover your expenses.</p>
+      <p><b>How Much?</b> The amount of the advance will depend on a number of factors, such as distance between the old and new permanent duty stations weight of household goods, member's grade, whether the member is using their government travel card on their <%= abbr_tag('pcs') %>, etc. Members and dependent travelers can get up to 80% of per diem and mileage allowance f the government travel card is not being used, <%= abbr_tag('dla') %> equal to 100% of the anticipated entitlement, and <%= abbr_tag('ppm') %>/<%= abbr_tag('dity') %> moves get an amount calculated by their transportation office. When possible, use your government travel card during your <%= abbr_tag('pcs') %>. You can find out more about the Government Travel Card <%= link_to 'here', 'https://www.citibank.com/tts/sa/dod/2016/q1/index.html', class: 'usa-external_link' %>.</p>
+      <p><b>Head’s Up:</b> You have to pay back your Advance after your move. It will come out of your paychecks over the next year.</p>
+      <p>When you file your final voucher, the Advance you took out will be paid first before you receive the remaining reimbursement.</p>
+    </div>
```

Comment:
For whatever text is shared between the military and civilian pages, I make the same copy-editing suggestions.